docs/src/analysis/global_sensitivity.md

```diff
@@ -141,22 +148,24 @@ t = collect(range(0, stop=10, length=200))
 For Morris Method
 
 ```julia
-m = DiffEqSensitivity.morris_sensitivity(prob,Tsit5(),t,[[1,5],[0.5,5]],[10,10],len_trajectory=1500,total_num_trajectory=1000,num_trajectory=150)
+f1 = function (p)
+        prob1 = remake(prob;p=p)
+        Array(solve(prob1,Tsit5();saveat=t))
+    end
```

Comment:
this example is still a little weird, since the global sensitivity at every point in a time series isn't really what a lot of people would go for.